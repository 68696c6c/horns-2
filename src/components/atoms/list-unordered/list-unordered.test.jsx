/* global describe, it, expect */
import React from 'react'

import { render } from 'setup-test'

import ListUnordered from '.'

describe('ListUnordered', () => {
  it('should render as default', () => {
    const { container } = render(
      <ListUnordered>
        <li>example</li>
        <li>example</li>
      </ListUnordered>
    )
    expect(container).toMatchSnapshot()
  })
})
