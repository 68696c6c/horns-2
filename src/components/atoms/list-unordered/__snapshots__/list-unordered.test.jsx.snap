// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListUnordered should render as default 1`] = `
.emotion-0 {
  grid-gap: 4px;
  display: grid;
  margin-top: 0px;
  margin-bottom: 0px;
  margin-left: 0px;
  margin-right: 0px;
  font-family: Helvetica,sans-serif;
  font-size: medium;
  font-style: normal;
  font-weight: 400;
  line-height: 1.6em;
  grid-template-rows: auto;
  padding-left: 32px;
}

<div>
  <ul
    class="emotion-0"
    type="disc"
  >
    <li>
      example
    </li>
    <li>
      example
    </li>
  </ul>
</div>
`;
